auth0/src/main/java/com/auth0/android/Auth0.java

```diff
@@ -183,20 +182,11 @@ public int getWriteTimeoutInSeconds() {
      * Setter for the user agent info to send in every request to Auth0.
      *
      * @param auth0UserAgent to send in every request to Auth0.
-     * @see #doNotSendAuth0UserAgent()
      */
     public void setAuth0UserAgent(@Nullable Auth0UserAgent auth0UserAgent) {
```

Comment:
Let's make the param non-null

```suggestion
    public void setAuth0UserAgent(@NonNull Auth0UserAgent auth0UserAgent) {
```

Reply:
Updated in new commit 👍